docs/articles/documentation/reference/testcafe-api/runner/run.md

```diff
@@ -33,6 +33,7 @@ Parameter         | Type    | Description
 `pageRequestTimeout` | Number | Time (in milliseconds) to wait for HTML pages. If the page isn't received within the specified period, an error is thrown. | `25000`
 `browserInitTimeout` | Number | Time (in milliseconds) for browsers to connect to TestCafe and report that they are ready to test. If one or more browsers fail to connect within this period, TestCafe throws an error. | `120000` for [local browsers](../../command-line-interface.md#local-browsers), `360000` for [remote browsers](../../command-line-interface.md#remote-browsers)
 `speed`           | Number  | Specifies the test execution speed. A number between `1` (fastest) and `0.01` (slowest). If an individual action's speed is also specified, the action speed setting overrides the test speed. | `1`
+`cache` | Boolean | If enabled, the TestCafe proxy caches webpage assets (such as stylesheets, images and scripts) for the webpages that it processes. The next time the proxy accesses the page, it pulls assets from its cache instead of requesting them from the server. | `false`
```

Comment:
Terms like `asset-heavy`, `run into`, and `pulls assets` are not suitable for technical docs. These are informal ways to describe things and are vague.

Reply:
```suggestion
`cache` | Boolean | If enabled, the TestCafe proxy caches webpage assets (such as stylesheets, images and scripts) for the webpages that it processes. The next time the proxy accesses the page, it loads assets from its cache instead of requesting them from the server. | `false`
```

Reply:
"Asset-heavy" is neither confusing or misleading. The phrase `a lot of assets` is even more vague because it doesn't account for things like sprites and request bundling. It is totally possible to have a lightweight web page that includes a lot of assets. There's no quantitative way to describe the increased network load from a high number of heavy remote assets - everyone has a different threshold for 'heavy'. 

As I see it, our main goal here is to get the point across without scaring the readers away. We aren't rewriting the Constitution and can afford to appeal to readers' emotions every now and then. 

Please offer a suggestion if you disagree :)